README.md

```diff
@@ -66,7 +66,11 @@ It can be used directly from the Terraform Registry like so:
 ```
 module "gke-cluster" {
   source  = "jetstack/gke-cluster/google"
+<<<<<<< HEAD
   version = "0.1.0"
+=======
+  version = "0.2.0-alpha1"
+>>>>>>> Porting to support Terraform v0.12
```

Comment:
Changing this to `version = "0.2.0-alpha1"` looks good to me! Could you tidy up the merge conflict?